models/data_maps.go

```diff
@@ -50,6 +50,7 @@ type DataMap struct {
 	NodeType       string    `json:"nodeType" db:"node_type"`
 	Message        string    `json:"message" db:"message"`
 	MsgID          string    `json:"msgId" db:"msg_id"`
+	MsgStatus      int       `json:"msgStatus" db:"msg_status"`
```

Comment:
In the next PR we could make this an enum like the Status field in this same model.